### ISSUE TITLE
[Bug] Cannot Execute Game after Compiling with Make on Mac OSX

### ISSUE THREAD

**Issue by acechoiboy** [Drewol/unnamed-sdvx-clone#282]
After compiling the program with make and then executing, the execution failed with the following error. I also noticed in the bin directory, it was looking for "Main.cfg: 1" even though there seems to be nothing there. 

I expected the program to run, but the following error message showed up.

```
xxx-MacBook-Pro:bin xxx$ ./usc-game 
[22:20:44][Info] Starting task "Application Setup"
[22:20:44][Info] Version: 0.4.0
[22:20:44][Info] Git commit: 2019-12-30_2456d6c
[22:20:44][Warning] Failed to open file for reading /Users/xxx/Desktop/unnamed-sdvx-clone/bin/./Main.cfg: 2
[22:20:44][Warning] Failed to load config file
[22:20:44][Warning] No joysticks found
[22:20:44][Warning] Failed to open file for reading /Users/xxx/Desktop/unnamed-sdvx-clone/bin/skins/Default/skin.cfg: 2
[22:20:44][Info] Starting task "Audio Init"
[22:20:44][Info] Audio driver [0]: coreaudio
[22:20:44][Info] Audio driver [1]: disk
[22:20:44][Info] Audio driver [2]: dummy
[22:20:44][Info] Using audio driver: coreaudio
[22:20:44][Info] Audio device [0]: Built-in Output
[22:20:44][Info] Audio device [1]: iShowU Audio Capture
[22:20:44][Info] Audio device [2]: Multi-Output Device
[22:20:44][Info] Finished task "Audio Init" in  190 ms
[22:20:44][Info] Starting task "GL Init"
[22:20:44][Info] OpenGL Version: 4.1 INTEL-12.10.14
[22:20:44][Info] OpenGL Shading Language Version: 4.10
[22:20:44][Info] OpenGL Renderer: Intel(R) Iris(TM) Plus Graphics 650
[22:20:44][Info] OpenGL Vendor: Intel Inc.
[22:20:44][Info] Finished task "GL Init" in  38 ms
[22:20:44][Info] Starting task "Discord RPC Init"
No bundle id found
[22:20:44][Info] Starting task "Check for updates"
[22:20:44][Info] Finished task "Discord RPC Init" in  0 ms
[22:20:44][Error] Shader program compile log for /Users/xxx/Desktop/unnamed-sdvx-clone/bin/skins/Default/shaders/gauge.fs: ERROR: 0:18: Invalid call of undeclared identifier 'texture2D'
ERROR: 0:19: Invalid call of undeclared identifier 'texture2D'
ERROR: 0:20: Use of undeclared identifier 'mask'
ERROR: 0:20: Use of undeclared identifier 'mask'
ERROR: 0:21: Use of undeclared identifier 'mask'
ERROR: 0:22: Use of undeclared identifier 'mask'
ERROR: 0:22: Use of undeclared identifier 'mask'
ERROR: 0:23: Use of undeclared identifier 'tex'
ERROR: 0:24: Use of undeclared identifier 'tex'
ERROR: 0:24: Use of undeclared identifier 'mask'

[22:20:44][Error] Failed to load fragment shader for material from /Users/xxx/Desktop/unnamed-sdvx-clone/bin/./skins/Default/shaders/gauge.fs
Segmentation fault: 11
```

**Comment by Drewol** [comment]
There's a mistake in the gauge.fs shader where it uses `texture2D` instead of `texture` so you just have to open that file in a text editor and remove the `2D` part

**Comment by Drewol** [comment]
This should be resolved in 5f349458c7b2329b4c932cfb7d95775fd6266aaa please confirm and close

**Comment by tarrball** [comment]
For what it's worth, I just compiled this on macOS 10.15.5 without issue.
